src/app/shared/components/checkout/basket-invoice-address-widget/basket-invoice-address-widget.component.ts

```diff
@@ -70,6 +67,26 @@ export class BasketInvoiceAddressWidgetComponent implements OnInit, OnDestroy {
       )
     );
 
+    this.fields = [
+      {
+        key: 'id',
+        type: 'ish-select-field',
+        templateOptions: {
+          fieldClass: 'col-12',
+          options: getAddressOptions(this.addresses$),
+          placeholder: this.emptyOptionLabel,
+        },
+        hooks: {
+          onInit: () => {
```

Comment:
I love to see some formly features but is there any upside to doing this subscription in the onInit hook instead of in the ngOnInit method? Because this way, it's needlessly intransparent and I've read about some people having issues with hooks.